it/mail/chrome/messenger/am-smime.properties

```diff
@@ -1,20 +1,20 @@
+NoSenderEncryptionCert=Hai specificato di crittografare questo messaggio, ma l'applicazione non ha potuto trovare il certificato di crittografia specificato nelle tue impostazioni account E-mail e Newsgroup, oppure il certificato è scaduto.
+MissingRecipientEncryptionCert=Hai specificato di crittografare questo messaggio, ma l'applicazione non ha potuto trovare un certificato di crittografia per %S.
+SignNoSenderEncryptionCert=Hai specificato di firmare digitalmente questo messaggio, ma l'applicazione non ha potuto trovare il certificato di crittografia da includere nel messaggio firmato, o il certificato è scaduto.
+ErrorCanNotEncrypt=Impossibile crittografare il messaggio. Controlla di possedere un certificato e-mail valido per ogni destinatario. Controlla anche che i certificati specificati nelle impostazioni account E-mail e Newsgroup siano validi e affidabili.
+ErrorCanNotSign=Impossibile firmare il messaggio. Controlla che i certificati specificati nelle impostazioni account E-mail e Newsgroup per questo account e-mail siano validi e affidabili.
 
 ## Strings used for in the prefs.
 prefPanel-smime=Sicurezza
-NoSigningCert=Gestione certificati non riesce ad individuare un certificato valido per la firma digitale dei propri messaggi.
-NoEncryptionCert=Gestione certificati non riesce ad individuare un certificato valido che gli altri possano usare per inviarci messaggi di posta cifrati.
+NoSigningCert=Il Gestore certificati non riesce ad individuare un certificato valido che può essere utilizzato per firmare digitalmente i tuoi messaggi.
+NoEncryptionCert=Il Gestore certificati non riesce ad individuare un certificato valido che gli altri possano utilizzare per inviarti messaggi e-mail crittografati.
 
-encryption_needCertWantSame=È necessario specificare un certificato che gli altri possano usare quando inviano qui messaggi cifrati. Si vuole usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
-encryption_wantSame=Usare lo stesso certificato per cifrare e decifrare i messaggi ricevuti?
-encryption_needCertWantToSelect=È necessario specificare un certificato che gli altri possano usare per rispondere con messaggi cifrati. Configurare un certificato di cifratura adesso?
-signing_needCertWantSame=Si deve specificare anche un certificato da usare per la firma digitale dei propri messaggi. Usare lo stesso certificato per firmare digitalmente i propri messaggi?
-signing_wantSame=Usare lo stesso certificato per firmare i propri messaggi?
-signing_needCertWantToSelect=Si deve specificare anche un certificato per la firma digitale dei propri messaggi. Configurare un certificato per firmare i messaggi digitalmente?
+encryption_needCertWantSame=Dovresti anche specificare un certificato che gli altri possano utilizzare quando t'inviano messaggi crittografati. Vuoi utilizzare lo stesso certificato per crittografare e decifrare i messaggi inviati a te?
```

Comment:
CORRECTIONS NEEDED. 

Elision and apocopation should be avoided when unnecessary. Please replace "t'" with "ti".